templates/taxi/car_list.html

```diff
@@ -1,16 +1,38 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list<a style="float: right" href="{% url 'taxi:car-create' %}">+</a></h1>
```

Comment:
Code Style issue: You should use 2 whitespaces indentations in your `.html` files. Here, you have used more than 2 spaces for indentation.